lib/passageidentity/error.rb

```diff
@@ -7,21 +7,44 @@ module Passage
   class PassageError < StandardError
     attr_reader :status_code, :error_code, :message, :status_text, :error
 
-    def initialize(message: nil, status_code: nil, body: nil)
-      super(message)
+    def initialize(message: nil, status_code: nil, body: {})
+      # Ensure body is a hash
+      body = {} if body.nil?
 
+      body = symbolize_keys(body)
```

Comment:
it looks like this is a [rails method](https://api.rubyonrails.org/classes/Hash.html#method-i-symbolize_keys). will that work in this sdk without us having to explicitly list it in our gemspec?

Reply:
This invokes a private method: https://github.com/passageidentity/passage-ruby/blob/0a8ed4d94cbb2b532d3b878d6e81ba82a302a619/lib/passageidentity/error.rb#L46-L48 which uses the [transform_keys](https://ruby-doc.org/core-2.5.8/Hash.html#method-i-transform_keys) method of Ruby's Hash class.

Reply:
oh whoops i missed that at the bottom.